src/hotspot/cpu/riscv/riscv_v.ad

```diff
@@ -2878,6 +2878,61 @@ instruct vclearArray_reg_reg(iRegL_R29 cnt, iRegP_R28 base, Universe dummy,
   ins_pipe(pipe_class_memory);
 %}
 
+// CompressBits of Long & Integer
+
+instruct compressBitsI(iRegINoSp dst, iRegIorL2I src, iRegIorL2I mask, vRegMask_V0 v0,
+                       vReg_V4 v4, vReg_V4 v5, vReg_V8 v8, vReg_V4 v9) %{
```

Comment:
Just noticed that this should be `vReg_V4 v4, vReg_V5 v5, vReg_V8 v8, vReg_V9 v9`.

Reply:
My bad, thanks for catching these typos!